src/sentry/models/activity.py

```diff
@@ -42,7 +42,10 @@ def get_activities_for_group(self, group: Group, num: int) -> Sequence[Activity]
             activity_qs = activity_qs.exclude(type=ActivityType.SET_PRIORITY.value)
         else:
             # Check if 'initial_priority' is available and the feature flag is on
-            initial_priority_value = group.get_event_metadata().get("initial_priority")
+            initial_priority_value = group.get_event_metadata().get("metadata", {}).get(
```

Comment:
Doesn't `get_event_metadata()` already return the metadata? I think this was right before

Reply:
whoops yep